scripts/moveDmp.pl

```diff
@@ -0,0 +1,246 @@
+##############################################################################
+#  Licensed Materials - Property of IBM
+#  Copyright (c) 2006, 2019 IBM Corp. All Rights Reserved.
+#  US Government Users Restricted Rights - Use, duplication or
+#  disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+##############################################################################
```

Comment:
Please change the license to be the same as https://github.com/AdoptOpenJDK/TKG/blob/master/scripts/resultsSum.pl

Reply:
> Please change the license to be the same as https://github.com/AdoptOpenJDK/TKG/blob/master/scripts/resultsSum.pl

Sure. Thanks for the link~:)